Makefile

```diff
@@ -4,12 +4,14 @@
 # Begin OS detection
 ifeq ($(OS),Windows_NT) # is Windows_NT on XP, 2000, 7, Vista, 10...
     export OPERATING_SYSTEM := Windows
-	export RUST_TARGET ?= "x86_64-unknown-windows-msvc"
```

Comment:
@Hoverbear do you know why these `RUST_TARGET` vars were here? I couldn't find them referenced anywhere in here or by `cross`, `rustc`, or `cargo`.

Reply:
Hmmm it think they are unused now. It may have been related to support for both gnu and msvc on Windows.